File: python/fastapi-sqlalchemy/schemas/breed.py
from sqlalchemy import Column, Integer, String
from db.db import Base
from sqlalchemy.orm import relationship


class Breed(Base):
    __tablename__ = "razas"

    id_breed = Column(
        name="codigoraza",
        type_=Integer,
        autoincrement="auto",
        nullable=False,
        primary_key=True,
    )
    breed = Column(name="raza", type_=String(length=128))

    pets = relationship("Pet", back_populates="breed")

File: javascript/express-pet-shelter/models/entities/breed.js
import { DataTypes } from "sequelize";
import { sequelize } from "../../db/db.js";

export const Breed = sequelize.define("razas", {
  idBreed: {
    type: DataTypes.INTEGER,
    field: "codigoraza",
  },
  breed: {
    type: DataTypes.STRING,
    field: "raza",
  },
});
